Automattic/wp-calypso
# Performance Profiler: update email alert banner

Closes https://github.com/Automattic/dotcom-forge/issues/8865
Closes https://github.com/Automattic/dotcom-forge/issues/8863

## Proposed Changes

* Update email banner styling
* Update copy and make it different for logged-in and logged-out users

## Why are these changes being made?


Feedback raised in:
* https://github.com/Automattic/dotcom-forge/issues/8863
* https://github.com/Automattic/dotcom-forge/issues/8865

## Testing Instructions



* Go to `/speed-test` and start a new test, or go to a results page eg. `/speed-test-tool?url=https%3A%2F%2Fwordpress.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzA1N30.7KXaBL1qcm0y0GqvM8ymJ0pbST9ls6P09_hri8EaKSI`
* Check the styling of the email alerts banner against Figma (Iix1FFHIQdfECIXn5JKT5s-fi-4259_34052), and check the copy in logged-in and logged-out states

| State | Mobile | Desktop |
|--------|--------|--------|
| Logged-in | ![CleanShot 2024-09-04 at 12 36 29@2x](https://github.com/user-attachments/assets/d9da3a4b-b000-456f-b42b-e8956dcf9ff2) | ![CleanShot 2024-09-03 at 16 11 39@2x](https://github.com/user-attachments/assets/bb6d65af-565d-4d18-abe4-23bd8a7c98e2) |
| Logged-out | ![CleanShot 2024-09-04 at 12 37 19@2x](https://github.com/user-attachments/assets/8319badd-b236-47f9-847c-ba19012fbb75) | ![CleanShot 2024-09-03 at 16 11 51@2x](https://github.com/user-attachments/assets/de2f362e-bace-4131-8115-002a802f6206) | 

## Pre-merge Checklist
